### ISSUE TITLE
Mismatch between expelliarmus and metavision

### ISSUE THREAD

**Issue by stevenabreu7** [fabhertz95/expelliarmus#6]
I loaded the same .raw file from a prophesee camera (evt2 format) with expelliarmus and metavision and I get very different numpy arrays from it. Here is the code I used for expelliarmus:
```python
from expelliarmus import Wizard
import numpy as np

wizard = Wizard(encoding="evt2", fpath="A1.raw")
arr = wizard.read()
np.save("A1.npy", arr)
```
And the code using prophesee's meta vision library:
```python
from metavision_core.event_io import EventsIterator
import numpy as np

ev_it = EventsIterator("A1.raw")
arr = np.concatenate([ev for ev in ev_it])
np.save("A1.npy", arr)
```
Here is a histogram of the events over time for both approaches (for eight different data files, top is meta vision, bottom is expelliarmus):
![hist_numpy](https://user-images.githubusercontent.com/19389742/204767530-726d2718-29e5-41d1-8ada-73102866dde3.png)
![hist_numpy_wiz](https://user-images.githubusercontent.com/19389742/204767571-8e4d01d9-fd69-4ba6-996e-9a9c4076e149.png)

Looking at the numpy data, I cannot easily see what went wrong. What's curious is that the first ~40 entries have timestamps <50, and then it jumps to a timestamp of >3e9. 

Any idea what could have gone wrong there?

**Comment by fabhertz95** [comment]
Uhm I think this is a bug inside `expelliarmus`, honestly. I will look into it immediately. Does it happen also for EVT3? I don't know if you have access to this kind of format.

**Comment by stevenabreu7** [comment]
Yeah unfortunately I don't have a EVT3 file available so I can't check. Thanks for looking into it so quickly!

**Comment by fabhertz95** [comment]
Could you try with [this EVT2 RAW file](https://dataset.prophesee.ai/index.php/s/mwiILym2zD8ud2b/download) and see what you get? If you could pass me the resulting NPY file on Discord, I would be really grateful :) You can find the link in the README of the repository.
